recipes/newrelic/apm/php/redhat.yml

```diff
@@ -66,7 +66,7 @@ install:
           if [ "{{.NR_PHP_RESTART}}" = "y" ]; then
             echo -e "{{.YELLOW}}* Automatic process restart was selected.  This installation WILL restart the php-fpm or nginx process depending on the application architecture detected.{{.NOCOLOR}}"
           else
-            echo -e "{{.YELLOW}}* Manual process restart was selected.  This installation WILL NOT automatically restart the php-fpm, apache, or nginx process depending on the application architecture detected.  You will be prompted to restart manually in order to complete a successful installation of the PHP agent.{{.NOCOLOR}}"
+            echo -e "{{.YELLOW}}* Manual process restart was selected.  This installation WILL NOT automatically restart the php-fpm or nginx process depending on the application architecture detected.  You will be prompted to restart manually in order to complete a successful installation of the PHP agent.{{.NOCOLOR}}"
```

Comment:
Was removing `apache` from the list intentional?

Reply:
Looks like `apache` is also missing from the list in the debian recipe.

Reply:
Not sure how it got removed.  I'll readd it.